pkg/compliance/rule.go

```diff
@@ -15,8 +15,9 @@ type Rule struct {
 
 // Scope defines when a rule can be run based on observed properties of the environment
 type Scope struct {
-	Docker     bool               `yaml:"docker"`
-	Kubernetes []KubeNodeSelector `yaml:"kubernetes,omitempty"`
+	Docker            bool               `yaml:"docker"`
+	Kubernetes        []KubeNodeSelector `yaml:"kubernetes,omitempty"`
+	KubernetesCluster bool               `yaml:"kubernetesCluster"`
```

Comment:
Should we stick with `snake_case` as in other places in existing code? Any disadvantages you see? We should pick one or the other not both.

Reply:
I'm personally fine with both. Aligned everything to `snake_case`

Reply:
In fact we already have camelCase everywhere in our YAML: `File.PathFrom`, `Command.maxOutputSize`.

The only place we use snake_case is in attributes we send to DD. It's fine as it's internal, but if we align something I'd align to camelCase.